test/iterators.jl

```diff
@@ -22,6 +22,22 @@ let z = zip(1:2, 3:4, 5:6)
     @test eltype(z) == Tuple{Int,Int,Int}
 end
 
+# unzip
+let z = zip(1:2)
+    @test unzip(z) == 1:2
```

Comment:
Since `unzip` always returns a `Tuple`, the right hand side of this test should be `(1:2,)`